pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -591,7 +592,8 @@ private void deleteSegmentsInternal(String tableNameWithType, List<String> segme
   @ApiOperation(value = "Get the server metadata for all table segments", notes = "Get the server metadata for all table segments")
   public String getServerMetadata(
       @ApiParam(value = "Name of the table", required = true) @PathParam("tableName") String tableName,
-      @ApiParam(value = "OFFLINE|REALTIME") @QueryParam("type") String tableTypeStr) {
+      @ApiParam(value = "OFFLINE|REALTIME") @QueryParam("type") String tableTypeStr,
+      @ApiParam(value = "Columns name", allowMultiple = true) @QueryParam("columns") @DefaultValue("") List<String> columns) {
```

Comment:
```suggestion
      @ApiParam(value = "Column name", allowMultiple = true) @QueryParam("columns") @DefaultValue("") List<String> columns) {
```

Reply:
IIUC, this parameter can be specified multiple times, right? Not sure how it will show up in swagger. Or, is it that we have a comma-separated column list? In that case, I would suggest "Column names (comma separated)"

Reply:
Not a comma-separated column list, but can be specified multiple times, since `allowMultiple = true`, a example query looks like:
```
curl http://localhost:18998/segments/mytable/metadata?columns=AirlineID&columns=ArrTime&columns=ArrTimeBlk
```